tests/jballerina-unit-test/src/test/resources/test-src/statements/expression/expression-stmt-negative.bal

```diff
@@ -15,3 +15,18 @@ function test2(){
     bar();
     x;
 }
+
+function baz() returns error|int|() => ();
```

Comment:
Having trouble reproducing this issue. Which ballerina version gives the NPE for this test case?